client/src/components/Search/Search.js

```diff
@@ -11,11 +11,36 @@ import SearchResults from './SearchResults/SearchResults';
  */
 export default class Search extends React.Component {
 
+    handlePageChange(page) {
+        const { onPageChange } = this.props;
+        onPageChange(page);
+    }
+
+    /**
+     * Whenever the monuments list changes (in length or at least one id has changed), scroll the search results
+     * up to the top, so that we see the first of the new results instead of wherever we might have been
+     * scrolled to before
+     */
+    componentDidUpdate(prevProps, prevState, snapshot) {
```

Comment:
What is the `snapshot` param for?

Reply:
Autfilled by IntelliJ using the function signature. Here's the documentation on it but it's something we will probably never use https://reactjs.org/docs/react-component.html#componentdidupdate